src/index.js

```diff
@@ -140,6 +144,7 @@ async function executeActions(req, context, params) {
         });
       } else {
         const url = appendURLParams(resolver.createURL(action), invokeParams);
+        log.debug(`calculated URL: ${url.replace(/\?.*$/, '?…')}`);
```

Comment:
suggest to add the same format as the ones above, eg:

```suggestion
        log.debug(`[${invokeInfo.idx}] calculated URL: ${url.replace(/\?.*$/, '?…')}`);
```

Reply:
I've removed it entirely, as the problem was somewhere else.